{{cookiecutter.project_slug}}/.neuro/live.yml

```diff
@@ -0,0 +1,157 @@
+kind: live
+title: {{ cookiecutter.project_name }}
+id: PROJECT_SLUG_WITH_UNDERSCORES_PLACEHOLDER
+
+defaults:
+  preset: gpu-small-p
+  life_span: 1d
+  tags:
+    - kind:flow
```

Comment:
Do we really need this one? Flow jobs already have many tags, containing the project id and kind of jobs (live).